oasislmf/pytools/getmodel/manager.py

```diff
@@ -312,12 +313,15 @@ def get_vulns(static_path, vuln_dict, num_intensity_bins, ignore_file_type=set()
                 num_damage_bins = header[0]
             if "vulnerability.idx" in static_path:
                 logger.debug(f"loading {os.path.join(static_path, 'vulnerability.idx')}")
-                vulns_bin = np.memmap(os.path.join(static_path, "vulnerability.bin"), dtype=VulnerabilityRow, offset=4, mode='r')
-                vulns_idx_bin = np.memmap(os.path.join(static_path, "vulnerability.idx"), dtype=VulnerabilityIndex, mode='r')
+                vulns_bin = np.memmap(os.path.join(static_path, "vulnerability.bin"),
```

Comment:
code is identical, I wouldn't reformat here as it respect our line length rule

Reply:
I agree, it must have been formatted when line length was the default 120. I'll revert it to one-line, it should fit in 150 characters

Reply:
done